src/app/edit/edit.component.ts

```diff
@@ -17,19 +18,33 @@ export class EditComponent implements OnInit {
   date:any;
   message='';
   editData:any
+  index:any;
+  address:any = [];
+  data:any
   
   constructor(private router:Router,private service: ServicesService) { }
 
   ngOnInit(): void {
+    this.service.getData().subscribe(result=>{
+      console.log(result.results)
+      console.log((result.results[1].business_status))
+      for(let i = 0; i<result.results.length ; i++){
+        if(result.results[i].plus_code){
+          this.address.push(result.results[i].plus_code.compound_code)
+        }
+      }
+    })
+    
     this.createUser = new FormGroup({
-      'eventName' : new FormControl('',[Validators.required,Validators.pattern('^[a-zA-Z ]*$'),Validators.maxLength(100)]),
+      'eventName' : new FormControl('',[Validators.required,Validators.pattern('^[a-zA-Z ]*$'),Validators.maxLength(100),this.blankSpace.bind(this)]),
```

Comment:
Make a constants file and add all regex at one place in order to make code reusable